es/07_1_Creando_una_Transaccion_Bitcoin_Parcialmente_Firmada.md

```diff
@@ -490,15 +490,15 @@ ea73a631b456d2b041ed73bf5767946408c6ff067716929a68ecda2e3e4de6d3
 
 Al crear el software `bitcoin-cli`, lo más probable es que cumpla las cinco funciones principales de los PSBT con `createpsbt`, `walletprocesspsbt` y `finalizepsbt`. Esto es lo que parece ese flujo:
 
-![](images/psbt-roles-for-cli-1.png)
+![](../images/psbt-roles-for-cli-1.png)
 
 Si elige usar el atajo de `walletcreatefundedpsbt`, así es como se ve en su lugar:
 
 ![](images/psbt-roles-for-cli-2.png)
```

Comment:
`![](../images/psbt-roles-for-cli-2.png)`

Reply:
sorry I missed that one, great attention to detail!